scout/parse/variant/conservation.py

```diff
@@ -1,37 +1,61 @@
+import logging
+
 from scout.constants import CONSERVATION
 import numbers
 
-def parse_conservations(variant):
+LOG = logging.getLogger(__name__)
+
+def parse_conservations(variant, vep_header=[]):
     """Parse the conservation predictors
 
         Args:
             variant(dict): A variant dictionary
+            vep_header(list): CSQ field might contain conservation annotations
 
         Returns:
             conservations(dict): A dictionary with the conservations
     """
     conservations = {}
 
-    conservations['gerp'] = parse_conservation(
-                                            variant,
-                                            'dbNSFP_GERP___RS'
-                                        )
-    conservations['phast'] = parse_conservation(
-                                            variant,
-                                            'dbNSFP_phastCons100way_vertebrate'
-                                        )
-    conservations['phylop'] = parse_conservation(
-                                            variant,
-                                            'dbNSFP_phyloP100way_vertebrate'
-                                        )
+    conservation_keys = {
+        'info_keys' : {
+            'gerp' : 'dbNSFP_GERP___RS',
+            'phast' : 'dbNSFP_phastCons100way_vertebrate',
+            'phylop' : 'dbNSFP_phyloP100way_vertebrate'
+        },
+        'csq_keys' : {
+            'gerp' : 'GERP++_RS',
+            'phast' : 'phastCons100way_vertebrate',
+            'phylop' : 'phyloP100way_vertebrate'
+        }
+    }
+    vep_info = variant.INFO.get('CSQ')
+
+    # if CSQ field contains any conservation key specified in conservation_keys['csq_keys']
+    if any(elem in list(conservation_keys['csq_keys'].values()) for elem in vep_header):
```

Comment:
This will slow down the code. For every entry in the vep header it will create a list, for every variant in the VCF. I would rewrite this and check the quickest check first. We can look at it together tomorrow if you like.

Reply:
Right, I'll try write a solution, otherwise together, yes, thanks!

Reply:
I've made it faster!